src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final String color = colorSupplier.getRandomColor();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(FIGURE_COUNT);
+        return switch (figureType) {
+            case 0 -> getIsoscelesTrapezoid();
+            case 1 -> getSquare();
+            case 2 -> getRectangle();
+            case 3 -> getCircle();
+            default -> getRightTriangle();
+        };
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle("white", 10);
```

Comment:
Add WHITE to Color enum, create constant for DEFAULT_COLOR, also create 10 as a constant